Automattic/jetpack
# Add premium-content to payments introducer

#### Changes proposed in this Pull Request:
Builds upon work in #22491 to add the premium-content wpcom-specific block to the payments introducer

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






### Setup

wpcom: Patch sandbox as usual

Atomic: Use jetpack beta plugin to select this branch

### Test

  1.  Load gutenberg
  2. In the block inserter / you should now see 'Payments'
  3. Try inserting a 'Payment' block, you'll then be given a choice of Payments Button, Donations Form and Premium Content
  4. Insert one, it should insert as normal
  5. Repeat to confirm that they all work
    
### Jetpack env
 1. Build the js packages, e.g. by running `pnpm run build-extensions` from the `projects/plugins/jetpack/extensions` dir
 2. Test as above, however this time you should not be able to see Premium Content as an option on the block inserter or in the payments block introducer.
